src/pcapi/routes/shared/passwords.py

```diff
@@ -56,14 +56,13 @@ def post_for_password_token(body: ResetPasswordBodyModel) -> None:
         # Here we also return a 204 to prevent attacker from discovering which email exists in db
         return
 
-    generate_reset_token(user)
-    repository.save(user)
+    token = create_reset_password_token(user)
 
     is_not_pro_user = user.isBeneficiary
 
     if is_not_pro_user:
         try:
-            send_reset_password_email_to_user(user)
+            send_reset_password_email_to_user(user, token)
```

Comment:
```suggestion
            send_reset_password_email_to_user(user, token.value)
```
ou alors, on remplace le `token: str` de `send_reset_password_email_to_user` par `Token` et on applique le `.value` la-bas.